Scripts/build-derived-assets.sh

```diff
@@ -38,7 +38,14 @@ fi
 
 DERIVED_ASSETS="${DIRECTORY}/DerivedAssets.xcassets"
 DERIVED_ASSETS_BASE="${DIRECTORY}/DerivedAssetsBase.xcassets"
+
+# Assets can be overridden by a DerivedAssetsOverride.xcassets in ${DIRECTORY}, or
```

Comment:
This seems a little odd to me, but okay.  What is the purpose of it?

Reply:
The original workspace builds overrode derived assets, for customizing Loop icons and such, by the existence of files in a parent directory (where the xcworkspace lives). Tidepool builds had to be updated to use a prepare script because there were two different xcworkspaces, and copy appropriate files in, but this parent directory method is still the build method used by DIY workspace builds.